server.js

```diff
@@ -652,6 +652,30 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Flattr integration
+camp.route(/^\/flattr\/(.*)\.(svg|png|gif|jpg|json)$/,
```

Comment:
A flattr is a donation of sorts. `flattrs_user_count` is how many people have done it. It seems that you can flattr something once a month.

You fill your account with a certain amount of money and then you go around and flattr things, then at the end of the month your budget is split between all of the things.

Anyway, I can't really think of any clear name that is better.